README.md

```diff
@@ -3,3 +3,22 @@
 [![Build Status](https://travis-ci.org/exercism/arm64-assembly.svg?branch=master)](https://travis-ci.org/exercism/arm64-assembly)
 
 Exercism exercises in ARM64 Assembly.
+
+## Assembler
+
+GNU Assembler (gas) is used for this track.
+
+## Testing
+
+Unity is a simple unit testing framework for C used by this track.
+Using a framework written in C instead of assembly language is easier to understand.
+It also shows how parameters are passed when calling assembly routines from C.
+
+## Variant and Calling Conventions
+
+This is probably going to use the ARMv8.0-A profile which is a good default.
+This track is also probably going to use the ARM default calling convention to spare mentors from trouble.
+
+## Contributing Guide
+
+Please see the [contributing guide](https://github.com/exercism/x-api/blob/master/CONTRIBUTING.md#the-exercise-data)
```

Comment:
Reference links, as per [the Exercism markdown guide][1].

[1]: https://github.com/exercism/docs/blob/main/building/markdown/markdown.md